olp-cpp-sdk-dataservice-read/tests/VolatileLayerClientImplTest.cpp

```diff
@@ -554,8 +548,7 @@ TEST(VolatileLayerClientImplTest, PrefetchTiles) {
 
     ASSERT_NE(future.wait_for(kTimeout), std::future_status::timeout);
     read::PrefetchTilesResponse response = future.get();
-    ASSERT_TRUE(response.IsSuccessful()) << response.GetError().GetMessage();
-    ASSERT_TRUE(response.GetResult().empty());
+    ASSERT_FALSE(response.IsSuccessful()) << response.GetError().GetMessage();
```

Comment:
```suggestion
    ASSERT_FALSE(response.IsSuccessful());
```
Check will fail if response is successful. In this case error message is always empty.

Btw, why response should be failed? Min/max levels are default, so only tiles from request loaded.
If behavior for default levels changed, we must update doxygen too :-)

Reply:
There is no tiles to load, so request fails if it is empty. Now behavior the same if we received empty quad, or request had wrong root tile, so there is no children or parent on requested levels.

Reply:
I think, we must fix it than. Purpose of the test is to load only tiles from request.
It makes no sense, if quad tree doesn't contain tiles.